### PR TITLE
refactor: use notifier-specific screenshot filename

### LINKED ISSUE #69: Image names should respect what notifier they came from
currently all images are called `collectionImage`

ideas:
`petImage`
`levelImage` // this one could have a better name
`lootImage` 
`deathImage`
`slayerImage`
`questImage`
`speedrunImage`
`killCountImage`
`combatTaskImage`
`achievementDiaryImage`

### PR DESCRIPTION
**Pull request by iProdigy**
Closes #69

### CODE REVIEW

**Review by Felanbird — APPROVED** [review]
Functionality works as intended 👍 - tested: `clue, loot, and boss Kill Count`

File: src/main/java/dinkplugin/notifiers/LevelNotifier.java [hunk 1]
```diff
@@ -104,6 +104,7 @@ private void attemptNotify() {
         createMessage(DinkPluginConfig::levelSendImage, NotificationBody.builder()
             .content(fullNotification)
             .extra(new LevelNotificationData(lSkills, currentLevels))
+            .screenshotFile("levelUpImage.png")
```
Comment (pajlada) [inline src/main/java/dinkplugin/notifiers/LevelNotifier.java:107]: Should probably be `levelImage.png` to be consistent with other image names in that they don't include the verb "lootReceived" or "achievmentDiaryCompleted"

**Review by pajlada — CHANGES_REQUESTED** [review]

File: src/main/java/dinkplugin/notifiers/LevelNotifier.java (hunk 1 above)
Comment (Felanbird) [reply src/main/java/dinkplugin/notifiers/LevelNotifier.java]: `levelImage` was my original suggestion but it read weird to me so I mentioned using something better

**Review by pajlada — APPROVED** [review]
Looks good to me :+1: 
@Mm2PL in case you want a quick glance, should be fairly straight forward. Everything works as expected for discord webhooks & runelite-dink-api. Only difference now is that the discord webhook 'direct URL's have a nicer name

Old: https://cdn.discordapp.com/attachments/1039842327344193617/1046163573644206150/collectionImage.png
New: https://cdn.discordapp.com/attachments/1039842327344193617/1046165861565730816/levelUpImage.png (would be `levelImage.png in final release)

**Pull request merged**